apps/admin/src/app/pages/user/permission-list.tsx

```diff
@@ -145,7 +144,7 @@ export default function PermissionList() {
 
     return (
         <Page title='Permission'>
-            <Container maxWidth={false}>
+            <Container maxWidth='lg'>
```

Comment:
Why added maxWidth='lg', I think false will be fine

Reply:
fix container